### PR TITLE
add JAVA_OPTS to Dockerfile

### LINKED ISSUE #208: No JAVA_OPTS available
So I have JHipster Registries running with 20GB of memory. For a regular JHipster application it is already possible to set special JAVA_OPTS via environment

### PR DESCRIPTION
**Pull request by xetys**
fix #208

- Please make sure the below checklist is followed for Pull Requests.

- [ ] [Travis tests](https://travis-ci.org/jhipster/jhipster-registry/pull_requests) are green
- [ ] Tests are added where necessary
- [ ] Documentation is added/updated where necessary
- [X] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/jhipster-registry/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

File: src/main/docker/Dockerfile [hunk 1]
```diff
@@ -10,4 +11,4 @@ VOLUME /tmp
 EXPOSE 8761
 CMD echo "The application will start in ${JHIPSTER_SLEEP}s..." && \
     sleep ${JHIPSTER_SLEEP} && \
-    java -Djava.security.egd=file:/dev/./urandom -jar /app.war
+    java ${JAVA_OPTS} -Djava.security.egd=file:/dev/./urandom -jar /app.jar
```
Comment (deepu105) [inline src/main/docker/Dockerfile]: Arent we generating war file?

**Comment by pascalgrimaud** [comment]
Indeed, we will be consistent with the one, generated for JHipster app: https://github.com/jhipster/generator-jhipster/blob/master/generators/server/templates/src/main/docker/_Dockerfile

But we should update this one too https://github.com/jhipster/jhipster-registry/blob/master/Dockerfile
This one is used for Docker Hub

File: src/main/docker/Dockerfile (hunk 1 above)
Comment (xetys) [reply src/main/docker/Dockerfile]: ooops...

**Comment by xetys** [comment]
i will fix the file later this morning!

**Comment by pascalgrimaud** [comment]
Oh, no worry @xetys, I already work on this, in the train, this morning
See the PR https://github.com/jhipster/jhipster-registry/pull/211

**Comment by xetys** [comment]
Thanks! Sorry for my current dev speed, I know it's a bit low :/

Pascal Grimaud <notifications@github.com> schrieb am Mo., 27. Nov. 2017,
09:38:

**Pull request merged**